python/usb_console/simulation.py

```diff
@@ -68,6 +72,9 @@ def configure_sim(self):
         self.computer_state_follower, self.computer_state_leader = self.eng.initialize_computer_states('not_detumbled', nargout=2)
         self.main_state_trajectory = []
 
+        self.eng.workspace['const']['dt'] = 120e6  # Control cycle time = 120 ms = 120e6 ns
```

Comment:
@nhz2 I hope this discrepancy doesn't affect the sim results by a lot

Reply:
I don't think anything assumes it was 0.1s and the integrators will probably still be accurate enough.